src/main/resources/org/openstreetmap/atlas/sqlite/tables_v1.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE metadata (key text NOT NULL, value);
+CREATE TABLE dictionary (key integer PRIMARY KEY, value);
+CREATE TABLE node (id integer PRIMARY KEY, checksum text, shard text, geom blob NOT NULL);
+CREATE TABLE node_tags (id integer NOT NULL, key integer NOT NULL, value integer NOT NULL);
+CREATE TABLE edge (id integer PRIMARY KEY, checksum text, shard text, geom blob NOT NULL);
```

Comment:
I would recommend change the shard column to integer. You can generate shard id's by taking Zoom-X-Y and just removing the dashes and making it an integer. This will give you a performance gain if using it in your where clause especially if you have a index on it.

Reply:
Also for edges I would suggest storing start and end node identifiers.

Reply:
Yeah, totally. I was thinking about doing both things.